### PR TITLE
Update email validation regex to reject commas in local part of address

### PR DESCRIPTION
**Pull request by klssmith**
- Also added a test to check that email addresses with a comma in the
domain part are not validated.

### CODE REVIEW

File: tests/test_recipient_validation.py [hunk 1]
```diff
@@ -138,6 +138,7 @@
     'spaces-in-domain@dom ain.com',
     'underscores-in-domain@dom_ain.com',
     'pipe-in-domain@example.com|gov.uk',
+    'comma,in-domain@gov.uk',
```
Comment (quis) [inline tests/test_recipient_validation.py]: This is `comma,in-local` technically (but functionally it doesn’t make a difference).

Comment (klssmith) [reply tests/test_recipient_validation.py]: Thanks - now updated.

**Comment by quis** [comment]
Can you add a test for a comma in the domain part as well (and make it pass if it doesn’t already)?

**Comment by klssmith** [comment]
@quis I've added in a test for commas in the domain part - (commas in the domain were already considered invalid, but we had no test for it)

**Pull request merged**